README.md

```diff
@@ -20,14 +20,38 @@ The API uses [Django REST framework](http://www.django-rest-framework.org/)
 
 
 ## Installation - Git and repository setup
-
+### Setting up your Git user and cloning the repo
 ```bash
 $ git config --global core.autocrlf false
 $ git config --global user.name "<your github username>"
 $ git config --global user.email your.github@email.com
 $ git clone git@github.com:dotkom/onlineweb4.git
 $ cd onlineweb4
```

Comment:
As you see here, every bash-command is prepended with `$`, which is quite helpful in ensuring that the following statement is the command to input, so I would like for this notation to continue

Reply:
On the other hand, starting every line with `$` makes the commands harder to copy paste 📋

Reply:
Also, `--global` can be removed. Should we advise people to set this globally when this is a matter of preference and not necessary for the setup of OW4?

Reply:
I agree with FluidSense, it becomes harder to copy commands when using them to set up the repo

Reply:
The reason the config has been set globally is to also have the config for other dotkom repos. Also, maybe Git was less used in the curriculum previously, making it less of a conflict.